tools/TestFx/TestClientFactory.cs

```diff
@@ -42,13 +42,12 @@ public TestClientFactory(MockContext mockContext)
 
         public TClient CreateArmClient<TClient>(IAzureContext context, string endpoint) where TClient : Rest.ServiceClient<TClient>
         {
-            if (typeof(TClient) != typeof(GraphRbacManagementClient))
+            if (typeof(TClient).Name != typeof(GraphRbacManagementClient).Name)
             {
                 return _mockContext.GetServiceClient<TClient>();
             }
 
-            var graphClient = _mockContext.GetGraphServiceClient<GraphRbacManagementClient>();
-            graphClient.TenantID = context.Tenant.Id;
+            var graphClient = _mockContext.GetGraphServiceClient<TClient>();
             return graphClient as TClient;
```

Comment:
The only reason to use GraphRbacManagementClient in the code was to set the client TenantID. Since it not necessary - you may want to remove references of the type form the code including using detective.